docs/getting_started/index.rst

```diff
@@ -1,11 +1,42 @@
 Getting started
 ===============
 
+Wagtail is built on Django, so this document assumes you've already got the essentials installed. But if not, those essentials are:
+
+ * `Python <https://wiki.python.org/moin/BeginnersGuide/Download>`_
+ * `pip <https://pip.pypa.io/en/latest/installing.html>`_ (Note that pip is included by default with Python 2.7.9 and later and Python 3.4 and later)
+
+We'd also strongly recommend:
+
+ * `Virtualenv <https://virtualenv.pypa.io/en/latest/installation.html>`_
+
+
+With the above installed and enabled, the quickest way to install wagtail is::
+
+    pip install wagtail
+
```

Comment:
What are those installations issues? I've tried following these instructions myself on osx (python 2.7.5) and not seen any errors yet.